src/components/transport_manager/src/transport_adapter/threaded_socket_connection.cc

```diff
@@ -304,7 +304,7 @@ bool ThreadedSocketConnection::Receive() {
           logger_,
           "Received " << bytes_read << " bytes for connection " << this);
       ::protocol_handler::RawMessagePtr frame(
-          new protocol_handler::RawMessage(0, 0, buffer, bytes_read));
+          new protocol_handler::RawMessage(0, 0, buffer, bytes_read, false));
```

Comment:
Would it be easier to set a default param value of false for encryption_required in the RawMessage constructor header? At what point is the encryption set to true for these message objects? It looks like they are all set to false.

Reply:
Actually it was a specific change request to avoid default parameters during internal review. As for protection flag value, `RawMessage` can be constructed from protocol packet in `HandleSingleFrameMessage`,  where the protection flag can be`true`